{"nodes":[{"content":"Thrown when ADSI returns an error during an operation to update the store.","nodes":[{"pos":[0,74],"content":"Thrown when ADSI returns an error during an operation to update the store.","nodes":[{"content":"Thrown when ADSI returns an error during an operation to update the store.","pos":[0,74]}]}],"pos":[1539,1614],"yaml":true},{"content":"Instantiates a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class.","nodes":[{"pos":[0,133],"content":"Instantiates a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"&gt;&lt;/xref&gt;</ph> class.","source":"Instantiates a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class."}],"pos":[3723,3857],"yaml":true},{"content":"The default constructor initializes a new instance of the <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> class. The <xref:System.Exception.Message%2A> property is initialized to a system-supplied message that describes the error. This message takes into account the current system culture. The <xref:System.Exception.InnerException%2A> property is initialized to `null`.","nodes":[{"pos":[0,401],"content":"The default constructor initializes a new instance of the <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> class. The <xref:System.Exception.Message%2A> property is initialized to a system-supplied message that describes the error. This message takes into account the current system culture. The <xref:System.Exception.InnerException%2A> property is initialized to `null`.","nodes":[{"content":"The default constructor initializes a new instance of the <ph id=\"ph1\">&lt;xref:System.DirectoryServices.AccountManagement.PrincipalOperationException&gt;</ph> class.","pos":[0,142],"source":"The default constructor initializes a new instance of the <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> class."},{"content":"The <ph id=\"ph1\">&lt;xref:System.Exception.Message%2A&gt;</ph> property is initialized to a system-supplied message that describes the error.","pos":[143,260],"source":" The <xref:System.Exception.Message%2A> property is initialized to a system-supplied message that describes the error."},{"content":"This message takes into account the current system culture.","pos":[261,320]},{"content":"The <ph id=\"ph1\">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property is initialized to <ph id=\"ph2\">`null`</ph>.","pos":[321,401],"source":" The <xref:System.Exception.InnerException%2A> property is initialized to `null`."}]}],"pos":[3868,4270],"yaml":true,"extradata":"MT"},{"content":"Instantiates a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class with the specified error message.","nodes":[{"pos":[0,166],"content":"Instantiates a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"&gt;&lt;/xref&gt;</ph> class with the specified error message.","source":"Instantiates a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class with the specified error message."}],"pos":[5714,5881],"yaml":true},{"content":"The <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> instance is initialized with the <xref:System.Exception.Message%2A> property set to the value of `message`. If `message` is `null`, the <xref:System.Exception.Message%2A> property is initialized to a system-supplied message.","nodes":[{"pos":[0,306],"content":"The <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> instance is initialized with the <xref:System.Exception.Message%2A> property set to the value of `message`. If `message` is `null`, the <xref:System.Exception.Message%2A> property is initialized to a system-supplied message.","nodes":[{"content":"The <ph id=\"ph1\">&lt;xref:System.DirectoryServices.AccountManagement.PrincipalOperationException&gt;</ph> instance is initialized with the <ph id=\"ph2\">&lt;xref:System.Exception.Message%2A&gt;</ph> property set to the value of <ph id=\"ph3\">`message`</ph>.","pos":[0,189],"source":"The <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> instance is initialized with the <xref:System.Exception.Message%2A> property set to the value of `message`."},{"content":"If <ph id=\"ph1\">`message`</ph> is <ph id=\"ph2\">`null`</ph>, the <ph id=\"ph3\">&lt;xref:System.Exception.Message%2A&gt;</ph> property is initialized to a system-supplied message.","pos":[190,306],"source":" If `message` is `null`, the <xref:System.Exception.Message%2A> property is initialized to a system-supplied message."}]}],"pos":[5892,6199],"yaml":true,"extradata":"MT"},{"content":"The text of the error message.","nodes":[{"pos":[0,30],"content":"The text of the error message.","nodes":[{"content":"The text of the error message.","pos":[0,30]}]}],"pos":[6354,6385],"yaml":true},{"content":"Initializes a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class from the specified <xref href=\"System.Runtime.Serialization.SerializationInfo\"></xref> and <xref href=\"System.Runtime.Serialization.StreamingContext\"></xref> instances.","nodes":[{"pos":[0,300],"content":"Initializes a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"&gt;&lt;/xref&gt;</ph> class from the specified <ph id=\"ph2\">&lt;xref href=\"System.Runtime.Serialization.SerializationInfo\"&gt;&lt;/xref&gt;</ph> and <ph id=\"ph3\">&lt;xref href=\"System.Runtime.Serialization.StreamingContext\"&gt;&lt;/xref&gt;</ph> instances.","source":"Initializes a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class from the specified <xref href=\"System.Runtime.Serialization.SerializationInfo\"></xref> and <xref href=\"System.Runtime.Serialization.StreamingContext\"></xref> instances."}],"pos":[8072,8373],"yaml":true},{"content":"A <xref href=\"System.Runtime.Serialization.SerializationInfo\"></xref> that contains the information required to serialize the new <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref>","nodes":[{"pos":[0,221],"content":"A <ph id=\"ph1\">&lt;xref href=\"System.Runtime.Serialization.SerializationInfo\"&gt;&lt;/xref&gt;</ph> that contains the information required to serialize the new <ph id=\"ph2\">&lt;xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"&gt;&lt;/xref&gt;</ph>","source":"A <xref href=\"System.Runtime.Serialization.SerializationInfo\"></xref> that contains the information required to serialize the new <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref>"}],"pos":[8653,8875],"yaml":true},{"content":"A <xref href=\"System.Runtime.Serialization.StreamingContext\"></xref> that contains the source of the serialized stream that is associated with the new <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref>","nodes":[{"pos":[0,242],"content":"A <ph id=\"ph1\">&lt;xref href=\"System.Runtime.Serialization.StreamingContext\"&gt;&lt;/xref&gt;</ph> that contains the source of the serialized stream that is associated with the new <ph id=\"ph2\">&lt;xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"&gt;&lt;/xref&gt;</ph>","source":"A <xref href=\"System.Runtime.Serialization.StreamingContext\"></xref> that contains the source of the serialized stream that is associated with the new <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref>"}],"pos":[8970,9213],"yaml":true},{"content":"Instantiates a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class with the specified error message and specified nested exception.","nodes":[{"pos":[0,197],"content":"Instantiates a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"&gt;&lt;/xref&gt;</ph> class with the specified error message and specified nested exception.","source":"Instantiates a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class with the specified error message and specified nested exception."}],"pos":[10660,10858],"yaml":true},{"content":"The <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> instance is initialized with the <xref:System.Exception.Message%2A> property set to the value of `message` and the <xref:System.Exception.InnerException%2A> property set to the value of `innerException`. If `message` is `null`, the <xref:System.Exception.Message%2A> property is initialized to a system-supplied message. The <xref:System.Exception.InnerException%2A> is initialized to `null`.","nodes":[{"pos":[0,474],"content":"The <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> instance is initialized with the <xref:System.Exception.Message%2A> property set to the value of `message` and the <xref:System.Exception.InnerException%2A> property set to the value of `innerException`. If `message` is `null`, the <xref:System.Exception.Message%2A> property is initialized to a system-supplied message. The <xref:System.Exception.InnerException%2A> is initialized to `null`.","nodes":[{"content":"The <ph id=\"ph1\">&lt;xref:System.DirectoryServices.AccountManagement.PrincipalOperationException&gt;</ph> instance is initialized with the <ph id=\"ph2\">&lt;xref:System.Exception.Message%2A&gt;</ph> property set to the value of <ph id=\"ph3\">`message`</ph> and the <ph id=\"ph4\">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property set to the value of <ph id=\"ph5\">`innerException`</ph>.","pos":[0,285],"source":"The <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> instance is initialized with the <xref:System.Exception.Message%2A> property set to the value of `message` and the <xref:System.Exception.InnerException%2A> property set to the value of `innerException`."},{"content":"If <ph id=\"ph1\">`message`</ph> is <ph id=\"ph2\">`null`</ph>, the <ph id=\"ph3\">&lt;xref:System.Exception.Message%2A&gt;</ph> property is initialized to a system-supplied message.","pos":[286,402],"source":" If `message` is `null`, the <xref:System.Exception.Message%2A> property is initialized to a system-supplied message."},{"content":"The <ph id=\"ph1\">&lt;xref:System.Exception.InnerException%2A&gt;</ph> is initialized to <ph id=\"ph2\">`null`</ph>.","pos":[403,474],"source":" The <xref:System.Exception.InnerException%2A> is initialized to `null`."}]}],"pos":[10869,11344],"yaml":true,"extradata":"MT"},{"content":"The text of the error message.","nodes":[{"pos":[0,30],"content":"The text of the error message.","nodes":[{"content":"The text of the error message.","pos":[0,30]}]}],"pos":[11525,11556],"yaml":true},{"content":"A nested exception.","nodes":[{"pos":[0,19],"content":"A nested exception.","nodes":[{"content":"A nested exception.","pos":[0,19]}]}],"pos":[11629,11649],"yaml":true},{"content":"Instantiates a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class with the specified error message and specified error code.","nodes":[{"pos":[0,191],"content":"Instantiates a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"&gt;&lt;/xref&gt;</ph> class with the specified error message and specified error code.","source":"Instantiates a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class with the specified error message and specified error code."}],"pos":[13072,13264],"yaml":true},{"content":"The <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> instance is initialized with the <xref:System.Exception.Message%2A> property set to the value of `message` and the <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException.ErrorCode%2A> property set to the value of `errorCode`. If `message` is `null`, the <xref:System.Exception.Message%2A> property is initialized to a system-supplied message.","nodes":[{"pos":[0,446],"content":"The <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> instance is initialized with the <xref:System.Exception.Message%2A> property set to the value of `message` and the <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException.ErrorCode%2A> property set to the value of `errorCode`. If `message` is `null`, the <xref:System.Exception.Message%2A> property is initialized to a system-supplied message.","nodes":[{"content":"The <ph id=\"ph1\">&lt;xref:System.DirectoryServices.AccountManagement.PrincipalOperationException&gt;</ph> instance is initialized with the <ph id=\"ph2\">&lt;xref:System.Exception.Message%2A&gt;</ph> property set to the value of <ph id=\"ph3\">`message`</ph> and the <ph id=\"ph4\">&lt;xref:System.DirectoryServices.AccountManagement.PrincipalOperationException.ErrorCode%2A&gt;</ph> property set to the value of <ph id=\"ph5\">`errorCode`</ph>.","pos":[0,329],"source":"The <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> instance is initialized with the <xref:System.Exception.Message%2A> property set to the value of `message` and the <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException.ErrorCode%2A> property set to the value of `errorCode`."},{"content":"If <ph id=\"ph1\">`message`</ph> is <ph id=\"ph2\">`null`</ph>, the <ph id=\"ph3\">&lt;xref:System.Exception.Message%2A&gt;</ph> property is initialized to a system-supplied message.","pos":[330,446],"source":" If `message` is `null`, the <xref:System.Exception.Message%2A> property is initialized to a system-supplied message."}]}],"pos":[13275,13722],"yaml":true,"extradata":"MT"},{"content":"The text of the message.","nodes":[{"pos":[0,24],"content":"The text of the message.","nodes":[{"content":"The text of the message.","pos":[0,24]}]}],"pos":[13892,13917],"yaml":true},{"content":"An error code.","nodes":[{"pos":[0,14],"content":"An error code.","nodes":[{"content":"An error code.","pos":[0,14]}]}],"pos":[13981,13996],"yaml":true},{"content":"Instantiates a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class with the specified error message, the specified nested exception, and the specified error code.","nodes":[{"pos":[0,228],"content":"Instantiates a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"&gt;&lt;/xref&gt;</ph> class with the specified error message, the specified nested exception, and the specified error code.","source":"Instantiates a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class with the specified error message, the specified nested exception, and the specified error code."}],"pos":[15503,15732],"yaml":true},{"content":"The <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> instance is initialized with the <xref:System.Exception.Message%2A> property set to the value of `message` and the <xref:System.Exception.InnerException%2A> property set to the value of `innerException`, and the <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException.ErrorCode%2A> is set to `errorCode`. If `message` is `null`, the <xref:System.Exception.Message%2A> property is initialized to a system-supplied message. The <xref:System.Exception.InnerException%2A> is initialized to `null`.","nodes":[{"pos":[0,596],"content":"The <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> instance is initialized with the <xref:System.Exception.Message%2A> property set to the value of `message` and the <xref:System.Exception.InnerException%2A> property set to the value of `innerException`, and the <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException.ErrorCode%2A> is set to `errorCode`. If `message` is `null`, the <xref:System.Exception.Message%2A> property is initialized to a system-supplied message. The <xref:System.Exception.InnerException%2A> is initialized to `null`.","nodes":[{"content":"The <ph id=\"ph1\">&lt;xref:System.DirectoryServices.AccountManagement.PrincipalOperationException&gt;</ph> instance is initialized with the <ph id=\"ph2\">&lt;xref:System.Exception.Message%2A&gt;</ph> property set to the value of <ph id=\"ph3\">`message`</ph> and the <ph id=\"ph4\">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property set to the value of <ph id=\"ph5\">`innerException`</ph>, and the <ph id=\"ph6\">&lt;xref:System.DirectoryServices.AccountManagement.PrincipalOperationException.ErrorCode%2A&gt;</ph> is set to <ph id=\"ph7\">`errorCode`</ph>.","pos":[0,407],"source":"The <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> instance is initialized with the <xref:System.Exception.Message%2A> property set to the value of `message` and the <xref:System.Exception.InnerException%2A> property set to the value of `innerException`, and the <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException.ErrorCode%2A> is set to `errorCode`."},{"content":"If <ph id=\"ph1\">`message`</ph> is <ph id=\"ph2\">`null`</ph>, the <ph id=\"ph3\">&lt;xref:System.Exception.Message%2A&gt;</ph> property is initialized to a system-supplied message.","pos":[408,524],"source":" If `message` is `null`, the <xref:System.Exception.Message%2A> property is initialized to a system-supplied message."},{"content":"The <ph id=\"ph1\">&lt;xref:System.Exception.InnerException%2A&gt;</ph> is initialized to <ph id=\"ph2\">`null`</ph>.","pos":[525,596],"source":" The <xref:System.Exception.InnerException%2A> is initialized to `null`."}]}],"pos":[15743,16340],"yaml":true,"extradata":"MT"},{"content":"The text of the message.","nodes":[{"pos":[0,24],"content":"The text of the message.","nodes":[{"content":"The text of the message.","pos":[0,24]}]}],"pos":[16536,16561],"yaml":true},{"content":"A nested exception.","nodes":[{"pos":[0,19],"content":"A nested exception.","nodes":[{"content":"A nested exception.","pos":[0,19]}]}],"pos":[16634,16654],"yaml":true},{"content":"An error code.","nodes":[{"pos":[0,14],"content":"An error code.","nodes":[{"content":"An error code.","pos":[0,14]}]}],"pos":[16718,16733],"yaml":true},{"content":"Returns an integer error code.","nodes":[{"pos":[0,30],"content":"Returns an integer error code.","nodes":[{"content":"Returns an integer error code.","pos":[0,30]}]}],"pos":[17980,18011],"yaml":true},{"content":"Returns an `integer`.  \n  \n The system will return 0 (zero) in the exception error code property if no underlying error was received from a win32 API. Callers need to distinguish between 0 (zero) and a valid win32 error code.","nodes":[{"pos":[0,21],"content":"Returns an <ph id=\"ph1\">`integer`</ph>.","source":"Returns an `integer`."},{"pos":[28,225],"content":"The system will return 0 (zero) in the exception error code property if no underlying error was received from a win32 API. Callers need to distinguish between 0 (zero) and a valid win32 error code.","nodes":[{"content":"The system will return 0 (zero) in the exception error code property if no underlying error was received from a win32 API. Callers need to distinguish between 0 (zero) and a valid win32 error code.","pos":[0,197],"nodes":[{"content":"The system will return 0 (zero) in the exception error code property if no underlying error was received from a win32 API.","pos":[0,122]},{"content":"Callers need to distinguish between 0 (zero) and a valid win32 error code.","pos":[123,197]}]}]}],"pos":[18120,18350],"yaml":true},{"content":"Initializes a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class with serialized data.","nodes":[{"pos":[0,153],"content":"Initializes a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"&gt;&lt;/xref&gt;</ph> class with serialized data.","source":"Initializes a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class with serialized data."}],"pos":[20016,20170],"yaml":true},{"content":"This constructor is called during deserialization to reconstitute the exception object transmitted over a stream. For more information, see [XML and SOAP Serialization](~/docs/framework/serialization/xml-and-soap-serialization.md).","nodes":[{"pos":[0,231],"content":"This constructor is called during deserialization to reconstitute the exception object transmitted over a stream. For more information, see [XML and SOAP Serialization](~/docs/framework/serialization/xml-and-soap-serialization.md).","nodes":[{"content":"This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.","pos":[0,113]},{"content":"For more information, see <bpt id=\"p1\">[</bpt>XML and SOAP Serialization<ept id=\"p1\">](~/docs/framework/serialization/xml-and-soap-serialization.md)</ept>.","pos":[114,231],"source":" For more information, see [XML and SOAP Serialization](~/docs/framework/serialization/xml-and-soap-serialization.md)."}]}],"pos":[20181,20413],"yaml":true,"extradata":"MT"},{"content":"The object that holds the serialized object data.","nodes":[{"pos":[0,49],"content":"The object that holds the serialized object data.","nodes":[{"content":"The object that holds the serialized object data.","pos":[0,49]}]}],"pos":[20741,20791],"yaml":true},{"content":"The contextual information about the source or destination.","nodes":[{"pos":[0,59],"content":"The contextual information about the source or destination.","nodes":[{"content":"The contextual information about the source or destination.","pos":[0,59]}]}],"pos":[20886,20946],"yaml":true}],"content":"### YamlMime:ManagedReference\nitems:\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  commentId: T:System.DirectoryServices.AccountManagement.PrincipalOperationException\n  id: PrincipalOperationException\n  children:\n  - System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor\n  - System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.String)\n  - System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  - System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.String,System.Exception)\n  - System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.String,System.Int32)\n  - System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.String,System.Exception,System.Int32)\n  - System.DirectoryServices.AccountManagement.PrincipalOperationException.ErrorCode\n  - System.DirectoryServices.AccountManagement.PrincipalOperationException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  langs:\n  - csharp\n  name: PrincipalOperationException\n  nameWithType: PrincipalOperationException\n  fullName: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  type: Class\n  assemblies:\n  - System.DirectoryServices.AccountManagement\n  namespace: System.DirectoryServices.AccountManagement\n  summary: Thrown when ADSI returns an error during an operation to update the store.\n  syntax:\n    content: 'public class PrincipalOperationException : System.DirectoryServices.AccountManagement.PrincipalException'\n  inheritance:\n  - System.Object\n  - System.Exception\n  - System.SystemException\n  - System.DirectoryServices.AccountManagement.PrincipalException\n  implements: []\n  inheritedMembers:\n  - System.Exception.Data\n  - System.Exception.GetBaseException\n  - System.Exception.GetType\n  - System.Exception.HelpLink\n  - System.Exception.HResult\n  - System.Exception.InnerException\n  - System.Exception.Message\n  - System.Exception.SerializeObjectState\n  - System.Exception.Source\n  - System.Exception.StackTrace\n  - System.Exception.TargetSite\n  - System.Exception.ToString\n  - System.Object.Equals(System.Object)\n  - System.Object.Equals(System.Object,System.Object)\n  - System.Object.GetHashCode\n  - System.Object.MemberwiseClone\n  - System.Object.ReferenceEquals(System.Object,System.Object)\n  version:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  monikers:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.DirectoryServices.AccountManagement/PrincipalOperationException.xml\n  author: BrucePerlerMS\n  ms.author: bruceper\n  manager: mbaldwin\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor\n  commentId: M:System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor\n  id: '#ctor'\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  langs:\n  - csharp\n  name: PrincipalOperationException()\n  nameWithType: PrincipalOperationException.PrincipalOperationException()\n  fullName: PrincipalOperationException.PrincipalOperationException()\n  type: Constructor\n  assemblies:\n  - System.DirectoryServices.AccountManagement\n  namespace: System.DirectoryServices.AccountManagement\n  summary: Instantiates a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class.\n  remarks: The default constructor initializes a new instance of the <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> class. The <xref:System.Exception.Message%2A> property is initialized to a system-supplied message that describes the error. This message takes into account the current system culture. The <xref:System.Exception.InnerException%2A> property is initialized to `null`.\n  syntax:\n    content: public PrincipalOperationException ();\n    parameters: []\n  overload: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor*\n  exceptions: []\n  version:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  monikers:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.DirectoryServices.AccountManagement/PrincipalOperationException.xml\n  author: BrucePerlerMS\n  ms.author: bruceper\n  manager: mbaldwin\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.String)\n  commentId: M:System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.String)\n  id: '#ctor(System.String)'\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  langs:\n  - csharp\n  name: PrincipalOperationException(String)\n  nameWithType: PrincipalOperationException.PrincipalOperationException(String)\n  fullName: PrincipalOperationException.PrincipalOperationException(String)\n  type: Constructor\n  assemblies:\n  - System.DirectoryServices.AccountManagement\n  namespace: System.DirectoryServices.AccountManagement\n  summary: Instantiates a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class with the specified error message.\n  remarks: The <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> instance is initialized with the <xref:System.Exception.Message%2A> property set to the value of `message`. If `message` is `null`, the <xref:System.Exception.Message%2A> property is initialized to a system-supplied message.\n  syntax:\n    content: public PrincipalOperationException (string message);\n    parameters:\n    - id: message\n      type: System.String\n      description: The text of the error message.\n  overload: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor*\n  exceptions: []\n  version:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  monikers:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.DirectoryServices.AccountManagement/PrincipalOperationException.xml\n  author: BrucePerlerMS\n  ms.author: bruceper\n  manager: mbaldwin\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  commentId: M:System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  langs:\n  - csharp\n  name: PrincipalOperationException(SerializationInfo, StreamingContext)\n  nameWithType: PrincipalOperationException.PrincipalOperationException(SerializationInfo, StreamingContext)\n  fullName: PrincipalOperationException.PrincipalOperationException(SerializationInfo, StreamingContext)\n  type: Constructor\n  assemblies:\n  - System.DirectoryServices.AccountManagement\n  namespace: System.DirectoryServices.AccountManagement\n  summary: Initializes a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class from the specified <xref href=\"System.Runtime.Serialization.SerializationInfo\"></xref> and <xref href=\"System.Runtime.Serialization.StreamingContext\"></xref> instances.\n  syntax:\n    content: protected PrincipalOperationException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);\n    parameters:\n    - id: info\n      type: System.Runtime.Serialization.SerializationInfo\n      description: A <xref href=\"System.Runtime.Serialization.SerializationInfo\"></xref> that contains the information required to serialize the new <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref>\n    - id: context\n      type: System.Runtime.Serialization.StreamingContext\n      description: A <xref href=\"System.Runtime.Serialization.StreamingContext\"></xref> that contains the source of the serialized stream that is associated with the new <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref>\n  overload: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor*\n  exceptions: []\n  version:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  monikers:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.DirectoryServices.AccountManagement/PrincipalOperationException.xml\n  author: BrucePerlerMS\n  ms.author: bruceper\n  manager: mbaldwin\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.String,System.Exception)\n  commentId: M:System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.String,System.Exception)\n  id: '#ctor(System.String,System.Exception)'\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  langs:\n  - csharp\n  name: PrincipalOperationException(String, Exception)\n  nameWithType: PrincipalOperationException.PrincipalOperationException(String, Exception)\n  fullName: PrincipalOperationException.PrincipalOperationException(String, Exception)\n  type: Constructor\n  assemblies:\n  - System.DirectoryServices.AccountManagement\n  namespace: System.DirectoryServices.AccountManagement\n  summary: Instantiates a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class with the specified error message and specified nested exception.\n  remarks: The <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> instance is initialized with the <xref:System.Exception.Message%2A> property set to the value of `message` and the <xref:System.Exception.InnerException%2A> property set to the value of `innerException`. If `message` is `null`, the <xref:System.Exception.Message%2A> property is initialized to a system-supplied message. The <xref:System.Exception.InnerException%2A> is initialized to `null`.\n  syntax:\n    content: public PrincipalOperationException (string message, Exception innerException);\n    parameters:\n    - id: message\n      type: System.String\n      description: The text of the error message.\n    - id: innerException\n      type: System.Exception\n      description: A nested exception.\n  overload: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor*\n  exceptions: []\n  version:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  monikers:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.DirectoryServices.AccountManagement/PrincipalOperationException.xml\n  author: BrucePerlerMS\n  ms.author: bruceper\n  manager: mbaldwin\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.String,System.Int32)\n  commentId: M:System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.String,System.Int32)\n  id: '#ctor(System.String,System.Int32)'\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  langs:\n  - csharp\n  name: PrincipalOperationException(String, Int32)\n  nameWithType: PrincipalOperationException.PrincipalOperationException(String, Int32)\n  fullName: PrincipalOperationException.PrincipalOperationException(String, Int32)\n  type: Constructor\n  assemblies:\n  - System.DirectoryServices.AccountManagement\n  namespace: System.DirectoryServices.AccountManagement\n  summary: Instantiates a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class with the specified error message and specified error code.\n  remarks: The <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> instance is initialized with the <xref:System.Exception.Message%2A> property set to the value of `message` and the <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException.ErrorCode%2A> property set to the value of `errorCode`. If `message` is `null`, the <xref:System.Exception.Message%2A> property is initialized to a system-supplied message.\n  syntax:\n    content: public PrincipalOperationException (string message, int errorCode);\n    parameters:\n    - id: message\n      type: System.String\n      description: The text of the message.\n    - id: errorCode\n      type: System.Int32\n      description: An error code.\n  overload: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor*\n  exceptions: []\n  version:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  monikers:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.DirectoryServices.AccountManagement/PrincipalOperationException.xml\n  author: BrucePerlerMS\n  ms.author: bruceper\n  manager: mbaldwin\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.String,System.Exception,System.Int32)\n  commentId: M:System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.String,System.Exception,System.Int32)\n  id: '#ctor(System.String,System.Exception,System.Int32)'\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  langs:\n  - csharp\n  name: PrincipalOperationException(String, Exception, Int32)\n  nameWithType: PrincipalOperationException.PrincipalOperationException(String, Exception, Int32)\n  fullName: PrincipalOperationException.PrincipalOperationException(String, Exception, Int32)\n  type: Constructor\n  assemblies:\n  - System.DirectoryServices.AccountManagement\n  namespace: System.DirectoryServices.AccountManagement\n  summary: Instantiates a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class with the specified error message, the specified nested exception, and the specified error code.\n  remarks: The <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException> instance is initialized with the <xref:System.Exception.Message%2A> property set to the value of `message` and the <xref:System.Exception.InnerException%2A> property set to the value of `innerException`, and the <xref:System.DirectoryServices.AccountManagement.PrincipalOperationException.ErrorCode%2A> is set to `errorCode`. If `message` is `null`, the <xref:System.Exception.Message%2A> property is initialized to a system-supplied message. The <xref:System.Exception.InnerException%2A> is initialized to `null`.\n  syntax:\n    content: public PrincipalOperationException (string message, Exception innerException, int errorCode);\n    parameters:\n    - id: message\n      type: System.String\n      description: The text of the message.\n    - id: innerException\n      type: System.Exception\n      description: A nested exception.\n    - id: errorCode\n      type: System.Int32\n      description: An error code.\n  overload: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor*\n  exceptions: []\n  version:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  monikers:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.DirectoryServices.AccountManagement/PrincipalOperationException.xml\n  author: BrucePerlerMS\n  ms.author: bruceper\n  manager: mbaldwin\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.ErrorCode\n  commentId: P:System.DirectoryServices.AccountManagement.PrincipalOperationException.ErrorCode\n  id: ErrorCode\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  langs:\n  - csharp\n  name: ErrorCode\n  nameWithType: PrincipalOperationException.ErrorCode\n  fullName: PrincipalOperationException.ErrorCode\n  type: Property\n  assemblies:\n  - System.DirectoryServices.AccountManagement\n  namespace: System.DirectoryServices.AccountManagement\n  summary: Returns an integer error code.\n  syntax:\n    content: public int ErrorCode { get; }\n    return:\n      type: System.Int32\n      description: \"Returns an `integer`.  \\n  \\n The system will return 0 (zero) in the exception error code property if no underlying error was received from a win32 API. Callers need to distinguish between 0 (zero) and a valid win32 error code.\"\n  overload: System.DirectoryServices.AccountManagement.PrincipalOperationException.ErrorCode*\n  exceptions: []\n  version:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  monikers:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.DirectoryServices.AccountManagement/PrincipalOperationException.xml\n  author: BrucePerlerMS\n  ms.author: bruceper\n  manager: mbaldwin\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  commentId: M:System.DirectoryServices.AccountManagement.PrincipalOperationException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  id: GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  langs:\n  - csharp\n  name: GetObjectData(SerializationInfo, StreamingContext)\n  nameWithType: PrincipalOperationException.GetObjectData(SerializationInfo, StreamingContext)\n  fullName: PrincipalOperationException.GetObjectData(SerializationInfo, StreamingContext)\n  type: Method\n  assemblies:\n  - System.DirectoryServices.AccountManagement\n  namespace: System.DirectoryServices.AccountManagement\n  summary: Initializes a new instance of the <xref href=\"System.DirectoryServices.AccountManagement.PrincipalOperationException\"></xref> class with serialized data.\n  remarks: This constructor is called during deserialization to reconstitute the exception object transmitted over a stream. For more information, see [XML and SOAP Serialization](~/docs/framework/serialization/xml-and-soap-serialization.md).\n  syntax:\n    content: >-\n      [System.Security.SecurityCritical]\n\n      public override void GetObjectData (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);\n    parameters:\n    - id: info\n      type: System.Runtime.Serialization.SerializationInfo\n      description: The object that holds the serialized object data.\n    - id: context\n      type: System.Runtime.Serialization.StreamingContext\n      description: The contextual information about the source or destination.\n  overload: System.DirectoryServices.AccountManagement.PrincipalOperationException.GetObjectData*\n  exceptions: []\n  attributes:\n  - type: System.Security.SecurityCriticalAttribute\n  version:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  monikers:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.DirectoryServices.AccountManagement/PrincipalOperationException.xml\n  author: BrucePerlerMS\n  ms.author: bruceper\n  manager: mbaldwin\nreferences:\n- uid: System.DirectoryServices.AccountManagement.PrincipalException\n  parent: System.DirectoryServices.AccountManagement\n  isExternal: false\n  name: PrincipalException\n  nameWithType: PrincipalException\n  fullName: System.DirectoryServices.AccountManagement.PrincipalException\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  isExternal: false\n  name: PrincipalOperationException()\n  nameWithType: PrincipalOperationException.PrincipalOperationException()\n  fullName: PrincipalOperationException.PrincipalOperationException()\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.String)\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  isExternal: false\n  name: PrincipalOperationException(String)\n  nameWithType: PrincipalOperationException.PrincipalOperationException(String)\n  fullName: PrincipalOperationException.PrincipalOperationException(String)\n- uid: System.String\n  parent: System\n  isExternal: false\n  name: String\n  nameWithType: String\n  fullName: System.String\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  isExternal: false\n  name: PrincipalOperationException(SerializationInfo, StreamingContext)\n  nameWithType: PrincipalOperationException.PrincipalOperationException(SerializationInfo, StreamingContext)\n  fullName: PrincipalOperationException.PrincipalOperationException(SerializationInfo, StreamingContext)\n- uid: System.Runtime.Serialization.SerializationInfo\n  parent: System.Runtime.Serialization\n  isExternal: false\n  name: SerializationInfo\n  nameWithType: SerializationInfo\n  fullName: System.Runtime.Serialization.SerializationInfo\n- uid: System.Runtime.Serialization.StreamingContext\n  parent: System.Runtime.Serialization\n  isExternal: false\n  name: StreamingContext\n  nameWithType: StreamingContext\n  fullName: System.Runtime.Serialization.StreamingContext\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.String,System.Exception)\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  isExternal: false\n  name: PrincipalOperationException(String, Exception)\n  nameWithType: PrincipalOperationException.PrincipalOperationException(String, Exception)\n  fullName: PrincipalOperationException.PrincipalOperationException(String, Exception)\n- uid: System.Exception\n  parent: System\n  isExternal: false\n  name: Exception\n  nameWithType: Exception\n  fullName: System.Exception\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.String,System.Int32)\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  isExternal: false\n  name: PrincipalOperationException(String, Int32)\n  nameWithType: PrincipalOperationException.PrincipalOperationException(String, Int32)\n  fullName: PrincipalOperationException.PrincipalOperationException(String, Int32)\n- uid: System.Int32\n  parent: System\n  isExternal: false\n  name: Int32\n  nameWithType: Int32\n  fullName: System.Int32\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor(System.String,System.Exception,System.Int32)\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  isExternal: false\n  name: PrincipalOperationException(String, Exception, Int32)\n  nameWithType: PrincipalOperationException.PrincipalOperationException(String, Exception, Int32)\n  fullName: PrincipalOperationException.PrincipalOperationException(String, Exception, Int32)\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.ErrorCode\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  isExternal: false\n  name: ErrorCode\n  nameWithType: PrincipalOperationException.ErrorCode\n  fullName: PrincipalOperationException.ErrorCode\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  isExternal: false\n  name: GetObjectData(SerializationInfo, StreamingContext)\n  nameWithType: PrincipalOperationException.GetObjectData(SerializationInfo, StreamingContext)\n  fullName: PrincipalOperationException.GetObjectData(SerializationInfo, StreamingContext)\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.#ctor*\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  isExternal: false\n  name: PrincipalOperationException\n  nameWithType: PrincipalOperationException.PrincipalOperationException\n  fullName: PrincipalOperationException.PrincipalOperationException\n  monikers:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.DirectoryServices.AccountManagement/PrincipalOperationException.xml\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.ErrorCode*\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  isExternal: false\n  name: ErrorCode\n  nameWithType: PrincipalOperationException.ErrorCode\n  fullName: PrincipalOperationException.ErrorCode\n  monikers:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.DirectoryServices.AccountManagement/PrincipalOperationException.xml\n- uid: System.DirectoryServices.AccountManagement.PrincipalOperationException.GetObjectData*\n  parent: System.DirectoryServices.AccountManagement.PrincipalOperationException\n  isExternal: false\n  name: GetObjectData\n  nameWithType: PrincipalOperationException.GetObjectData\n  fullName: PrincipalOperationException.GetObjectData\n  monikers:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.DirectoryServices.AccountManagement/PrincipalOperationException.xml\n- uid: System.Object.Equals(System.Object)\n  parent: System.Object\n  isExternal: false\n  name: Equals(Object)\n  nameWithType: Object.Equals(Object)\n  fullName: Object.Equals(Object)\n- uid: System.Object.Equals(System.Object,System.Object)\n  parent: System.Object\n  isExternal: false\n  name: Equals(Object, Object)\n  nameWithType: Object.Equals(Object, Object)\n  fullName: Object.Equals(Object, Object)\n- uid: System.Object.GetHashCode\n  parent: System.Object\n  isExternal: false\n  name: GetHashCode()\n  nameWithType: Object.GetHashCode()\n  fullName: Object.GetHashCode()\n- uid: System.Exception.GetType\n  parent: System.Exception\n  isExternal: false\n  name: GetType()\n  nameWithType: Exception.GetType()\n  fullName: Exception.GetType()\n- uid: System.Object.MemberwiseClone\n  parent: System.Object\n  isExternal: false\n  name: MemberwiseClone()\n  nameWithType: Object.MemberwiseClone()\n  fullName: Object.MemberwiseClone()\n- uid: System.Object.ReferenceEquals(System.Object,System.Object)\n  parent: System.Object\n  isExternal: false\n  name: ReferenceEquals(Object, Object)\n  nameWithType: Object.ReferenceEquals(Object, Object)\n  fullName: Object.ReferenceEquals(Object, Object)\n- uid: System.Exception.ToString\n  parent: System.Exception\n  isExternal: false\n  name: ToString()\n  nameWithType: Exception.ToString()\n  fullName: Exception.ToString()\n- uid: System.Exception.Data\n  parent: System.Exception\n  isExternal: false\n  name: Data\n  nameWithType: Exception.Data\n  fullName: Exception.Data\n- uid: System.Exception.GetBaseException\n  parent: System.Exception\n  isExternal: false\n  name: GetBaseException()\n  nameWithType: Exception.GetBaseException()\n  fullName: Exception.GetBaseException()\n- uid: System.Exception.HelpLink\n  parent: System.Exception\n  isExternal: false\n  name: HelpLink\n  nameWithType: Exception.HelpLink\n  fullName: Exception.HelpLink\n- uid: System.Exception.HResult\n  parent: System.Exception\n  isExternal: false\n  name: HResult\n  nameWithType: Exception.HResult\n  fullName: Exception.HResult\n- uid: System.Exception.InnerException\n  parent: System.Exception\n  isExternal: false\n  name: InnerException\n  nameWithType: Exception.InnerException\n  fullName: Exception.InnerException\n- uid: System.Exception.Message\n  parent: System.Exception\n  isExternal: false\n  name: Message\n  nameWithType: Exception.Message\n  fullName: Exception.Message\n- uid: System.Exception.SerializeObjectState\n  parent: System.Exception\n  isExternal: false\n  name: SerializeObjectState\n  nameWithType: Exception.SerializeObjectState\n  fullName: Exception.SerializeObjectState\n- uid: System.Exception.Source\n  parent: System.Exception\n  isExternal: false\n  name: Source\n  nameWithType: Exception.Source\n  fullName: Exception.Source\n- uid: System.Exception.StackTrace\n  parent: System.Exception\n  isExternal: false\n  name: StackTrace\n  nameWithType: Exception.StackTrace\n  fullName: Exception.StackTrace\n- uid: System.Exception.TargetSite\n  parent: System.Exception\n  isExternal: false\n  name: TargetSite\n  nameWithType: Exception.TargetSite\n  fullName: Exception.TargetSite\n"}